crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -122,7 +125,7 @@ impl<T: ShaderType + WriteInto> StorageBuffer<T> {
         if self.capacity < size || self.changed {
             self.buffer = Some(device.create_buffer_with_data(&BufferInitDescriptor {
                 label: self.label.as_deref(),
-                usage: self.buffer_usage,
+                usage: BufferUsages::STORAGE | self.buffer_usage,
```

Comment:
Why do this every time if usages cannot be removed?

Reply:
@james7132 I see that you commented suggesting this change. If usages can only be added, not removed, then to me it makes sense that the required usages remain in the initial values and others can be added.

Reply:
Shall I revert the change?

Reply:
Oh shoot, missed that it was add only. This initialization change could probably be reverted. However, it should be reflected in the docs that setting the usages is additive and always includes the defaults.